pkg/anago/release.go

```diff
@@ -439,19 +440,29 @@ func (d *DefaultRelease) PushGitObjects() error {
 func (d *DefaultRelease) CreateAnnouncement() error {
 	// Buld the announcement options set
 	announceOpts := announce.NewOptions()
-	announceOpts.WithWorkDir(gitRoot)
-	announceOpts.WithTag(util.SemverToTagString(d.state.semverBuildVersion))
-	announceOpts.WithBranch(d.options.ReleaseBranch)
+
+	// Workdir is where the announce files will be saved
+	announceOpts.WithWorkDir(workspaceDir + "/src")
```

Comment:
Tiny nit: should we use `filepath.Join` here?

Reply:
Changed too, but I'm looking at the test failures and I think I'll have to move a bunch of the functionality here deeper into the announce package.

Reply:
I'm adding validation for the announce values here, but now the tests are broken. Specifically when trying to read the changelog html.

Perhaps this could be a good time to transform the announce package into the implementation + fakes pattern we've been using :thinking: